src/Compilers/Core/Portable/SpecialMember.cs

```diff
@@ -196,6 +196,8 @@ internal enum SpecialMember
 
         System_Runtime_CompilerServices_AsyncHelpers__AwaitAwaiter_TAwaiter,
         System_Runtime_CompilerServices_AsyncHelpers__UnsafeAwaitAwaiter_TAwaiter,
+        System_Runtime_CompilerServices_AsyncHelpers__HandleAsyncEntryPoint_Task,
+        System_Runtime_CompilerServices_AsyncHelpers__HandleAsyncEntryPoint_Task_Int32,
```

Comment:
Question not related to this code location: 
The "async Main" feature spec included details about codegen: [https://github.com/dotnet/csharplang/blob/main/proposals/csharp-7.1/async-main.md](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-7.1/async-main.md)
Do we need a corresponding spec change for this work? Should LDM be made aware?